### PR TITLE
Update first-steps.md

### PR DESCRIPTION
**Pull request by StackScribe**
It seems like we should briefly explain what the Bridge is here --I'm not sure that my definition is accurate, though.  Then we should explain what a sequence is before we tell them to click on it.  Also, Keptn needs to be capitalized and can't have apostrophe-s attached to it.

### CODE REVIEW

File: first-steps.md [hunk 1]
```diff
@@ -2,15 +2,30 @@
 
 # First Steps After Installation
 
-Head to the [keptn's bridge](http://localhost/bridge). Notice one project `helloworld` which has one stage `demo`. [Go to the sequence view](http://localhost/bridge/project/helloworld/sequence) for the project and click the sequence. the sequence has completed successfully and the log output is `Hello, world!`
+Head to the [Keptn Bridge](http://localhost/bridge), which is the UI from which you can observe and manage what Keptn executes.
+
+You should understand a bit of Keptn terminology:
+* Each Keptn project contains one or more `stages`, each of which corresponds to a major objective, such as delivery, remediation.
```
Comment (agardnerIT) [inline first-steps.md:8]: * Each Keptn project contains one or more `stages`, each of which corresponds to an environment, such as development or production.
* Each `stage` contains one or more `sequences`, each of which corresponds to major objective, such as delivery of an artifact or a rollback action.
* A sequence is a list of `tasks` to be executed sequentially.
  Most `tasks` describe a piece of work accomplished by the external tools you have configured,
  such as `evaluation` or `approval`.
* Stage, sequence and task names are arbitrary; you can can create your own. However Keptn ships with microservices preconfigured to listen for some task events. For example, the [lighthouse service](https://github.com/keptn/keptn/tree/master/lighthouse-service) listens for the `evaluation` task and executes quality evaluations.
* Keptn is unopinionated about how a task is actioned. You are free to adjust the default setup which, in effect, means bringing your own tools. You will see this as you follow this tutorial.

Comment (StackScribe) [reply first-steps.md:8]: Love it!  This is starting to read like an exercise that one could actually learn from!

Comment (StackScribe) [inline first-steps.md:8]:
```suggestion
```

Comment (StackScribe) [inline first-steps.md:8]:
```suggestion
* Each Keptn project contains one or more stages, each of which corresponds to an environment, such as development or production.
```

File: first-steps.md [hunk 2]
```diff
@@ -2,15 +2,29 @@
 
 # First Steps After Installation
 
-Head to the [keptn's bridge](http://localhost/bridge). Notice one project `helloworld` which has one stage `demo`. [Go to the sequence view](http://localhost/bridge/project/helloworld/sequence) for the project and click the sequence. the sequence has completed successfully and the log output is `Hello, world!`
+Head to the [Keptn Bridge](http://localhost/bridge), which is the UI from which you can observe and manage what Keptn executes.
+
+You should understand a bit of Keptn terminology:
+* Each `stage` contains one or more `sequences`, each of which corresponds to a step required to accomplish the objective.
```
Comment (StackScribe) [inline first-steps.md:8]:
```suggestion
* Each Keptn project contains one or more stages, each of which corresponds to an environment, such as development or production.
+ Each stage contains one or more sequences, each of which corresponds to major objective, such as delivery of an artifact or a rollback action.
```

File: first-steps.md [hunk 3]
```diff
@@ -2,15 +2,29 @@
...
+* A sequence is a list of `tasks` to be executed sequentially.
```
Comment (StackScribe) [inline first-steps.md:9]:
```suggestion
* A sequence is a list of tasks to be executed sequentially.
Most tasks describe a piece of work accomplished by the external tools you have configured,
such as evaluation or approval.
```

**Pull request merged**